src/components/SchemaSlotHelp/SchemaSlotHelp.tsx

```diff
@@ -9,16 +17,14 @@ const SchemaSlotHelp: React.FC<SchemaSlotHelpProps> = ({
   slot,
   containingObjectName = "record",
 }) => {
+  // Get any pattern information present in this slot definition.
+  const pattern = slot.pattern;
+
+  // Process other types of guidance present on this slot's specification.
   const guidanceParagraphs: string[] = [];
   if (slot.comments && slot.comments.length) {
     guidanceParagraphs.concat(...slot.comments);
   }
-  if (slot.pattern) {
-    guidanceParagraphs.push("Pattern as regular expression: " + slot.pattern);
-  }
-  if (slot.structured_pattern) {
-    guidanceParagraphs.push("Pattern hint: " + slot.structured_pattern.syntax);
-  }
```

Comment:
Ah! I didn't even realize that was in there in first place. I still think taking it out completely is the right move until we can be more sure that the `submission-schema` build process is correctly keeping `structured_pattern` and `pattern` in sync.